import.py

```diff
@@ -40,13 +41,41 @@ def main():
 
   first = True
   for f in os.listdir(dataDir):
-    if f[-4:].lower() == ".shp":
+    rasterFound = False
+    shapefileFound = False
+    if f[-4:].lower() == ".tif":
```

Comment:
Are rasters always a '.tif' or can they be some other image format?

Reply:
There are other formats, but I'm not yet sure if we'll be able to support them automatically or would have to build separate loading infrastructure for each.  So for right now I think we should just say we support GeoTIFF, which is the most common in general and the only one I've seen from Carson so far.